lib/hacluster.pm

```diff
@@ -121,6 +121,7 @@ sub ensure_process_running {
     my $timeout   = 30;
     my $starttime = time;
     my $ret       = undef;
+    $timeout *= get_var('TIMEOUT_SCALE', 1);
```

Comment:
Why didn't you directly do `my $timeout = 30 * get_var('TIMEOUT_SCALE', 1);`?

Reply:
You're correct. That looks better. Will change